lib/jxl.cmake

```diff
@@ -197,6 +197,8 @@ set(JPEGXL_INTERNAL_SOURCES_DEC
   jxl/render_pipeline/simple_render_pipeline.cc
   jxl/render_pipeline/simple_render_pipeline.h
   jxl/render_pipeline/test_render_pipeline_stages.h
+  jxl/render_pipeline/xyb_stage.cc
+  jxl/render_pipeline/xyb_stage.h
```

Comment:
It might make sense to use filenames of the form `stage_something` instead of `something_stage`, so they'll stay together when sorted.

Reply:
Doesn't make that much of a difference as they're all under render_pipeline, but done.